src/parquet_bridge.rs

```diff
@@ -96,6 +97,84 @@ impl From<Compression> for CompressionCodec {
     }
 }
 
+/// Defines the compression settings for writing a parquet file.
+#[derive(Debug, Eq, PartialEq, Hash, Clone, Copy)]
+pub enum CompressionEncode {
```

Comment:
In terms of how the compression structs should be named, I wondered whether `Compression` should be refactored to `CompressionDecode`?

Reply:
compression and encoding have their specific notation in parquet. I think it would be better if we would reserve the words "decode" and "encode" to things related to parquet encoding.

`CompressionOptions` seems the most consistent with `WriteOptions` that we already use.